ddocs/medic/_attachments/translations/messages-sw.properties

```diff
@@ -1150,4 +1150,5 @@ task.warning.home_birth =
 mrdt.disabled = 
 mrdt.verify = 
 contacts.visits.visits = 
-contacts.visits.count = 
\ No newline at end of file
+contacts.visits.count = 
+already\ taken = ime sha chukuliwa. 
```

Comment:
You should only add your translations in English. Translators will update the other languages from POEditor.